app/assets/stylesheets/user_show.scss

```diff
@@ -0,0 +1,109 @@
+@mixin shadow1 {
+  box-shadow: 1px 1px 1px #888888;
+}
+@mixin shadow2 {
+  box-shadow: 2px 2px 2px #888888;
+}
+@mixin shadow3 {
+  box-shadow: 3px 3px 3px #888888;
+}
+@mixin shadow4 {
+  box-shadow: 4px 4px 4px #888888;
+}
+
+.topcorner{
+   position:absolute;
+   top:-20px;
+   right:10px;
+  }
+
+  .score-messages-container {
+    margin: 0 10px 0 0;
+  }
+
+.score-username-container {
+  @include shadow1;
+  min-width: 150px;
```

Comment:
Properties should be ordered background-color, clear, float, height, margin-top, min-width, overflow, padding, text-align